.storybook/main.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
```

Comment:
There is no need to have support for `jsx`, `ts`, and `tsx` extensions. We will be using only `mdx` (for displaying tokens) and `js` (in `stories.js`). Too many allowed options encourage more messy project structure. It is good idea to restrict a bit.

Reply:
fixed